### DISCUSSION TITLE
Support offline docs with static code examples for Dash/devdocs.io

### DISCUSSION THREAD

**Discussion by mdkq** [fastapi/fastapi#6420] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

Right now I can get offline docs by downloading the docs folder, or by using Dash's github feature, but none of the code examples work since they are dynamic.

### Wanted Solution

Minimal solution would be to provide static code examples and a numerically organized docs folder (because the tutorial is out of order if you just download it as is), complete solution would provide docs for devdocs.io/Dash

### Wanted Code

```python
n/a
```

### Operating System

Other

### Operating System Details

n/a

### FastAPI Version

n/a

### Python Version

n/a

### Additional Context

Loving FastAPI so far. Thanks to everyone for your hard work

**Comment by tiangolo** [comment, marked as answer]
I assume you solved it, thanks for closing the issue 👍

If you want to build the docs to keep them offline, you can follow the instructions here: https://fastapi.tiangolo.com/contributing/#docs

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
